### DISCUSSION TITLE
How to Watch videos using an external player?

### DISCUSSION THREAD

**Discussion by Xelbayria** [FreeTubeApp/FreeTube#3408] [Q&A]
I installed Potplayer and wasn't able to figure out how to use it as external player. Yes I am aware of VLC, I don't want to use VLC because it lacks one feature that potplayer. 

Okay, I wasn't able to find any more detail on how to watch videos using external player. I've tried Potplayer (seeing it's not supported) and tried VLC.  I have no idea how to get freetube to pass the youtube-URL to the external player. I cannot find the key information on how to do it?   Was there hotkey for doing this or special action????

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @Xelbayria just to clarify for myself. Is this question only about Potplayer?

**Reply by Xelbayria** [reply]
> Hi @Xelbayria just to clarify for myself. Is this question only about Potplayer?

Yes, it is about potplayer.  i almost got it working BUT there are problems. Here's a good picture:
![image](https://user-images.githubusercontent.com/12866666/236700909-20a931d9-4825-4096-94d5-39d79c39c53e.png)

and

![image](https://user-images.githubusercontent.com/12866666/236700923-b595ba5c-36f0-4c6c-b0a8-6de7dad7c302.png)

You can see there is `--rate=1`. I am not sure how to solve that and it shouldn't be showing up. 

i remembered that VLC is using a rate of 1.. while potplayer is using a rate of 1.25 from other issues where people are talking about the difference of potplayer and VLC when using them as an external player. 

I also tried other options aside VLC. only VLC is the best one that could almost give me what I want using Potplayer. It's too bad about MPC-HC not having an support for video link via internet like youtube.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I'm going to close this in favor of https://github.com/FreeTubeApp/FreeTube/issues/3614. Please include any info u think is needed in that issue.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
inquires regarding external players migrated to https://github.com/FreeTubeApp/FreeTube/discussions/7789
